CHANGES.txt

```diff
@@ -56,6 +56,13 @@ Changes
 Fixes
 =====
 
+ - Throw proper exception when using the ``_raw`` inside the where clause
```

Comment:
I think `the` could be removed, or otherwise it should be extended to `using the _raw column`